runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -1208,9 +1207,13 @@ private void petListLookup(ChatMessage chatMessage, String message)
 		Pet[] pets = Pet.values();
 		for (Pet pet : pets)
 		{
-			if (playerPetList.contains(pet.getIconID()))
+			if (playerPetList.containsKey(pet.getIconID()))
 			{
-				responseBuilder.append(" ").img(modIconIdx + pet.ordinal());
+				Integer petQuantity = playerPetList.get(pet.getIconID());
+				for (int i = 0; i < petQuantity; i++)
+				{
+					responseBuilder.append(" ").img(modIconIdx + pet.ordinal());
+				}
```

Comment:
Duplicating the icon seems very weird to me. I think it would be much better implemented like so:
```java
responseBuilder.append(" ").img(modIconIdx + pet.ordinal());
int petQuantity = playerPetList.get(pet.getIconID());
if (petQuantity > 1)
{
	responseBuilder.append("x" + petQuantity);
}
```

Reply:
Originally I thought duplicating the pet icon would be a more pleasing user aesthetic. However, if this is not recommended, then I can change it